src/models/activityReportObjective.js

```diff
@@ -15,6 +15,14 @@ export default (sequelize, DataTypes) => {
       ActivityReportObjective.hasMany(models.ActivityReportObjectiveFile, { foreignKey: 'activityReportObjectiveId', as: 'activityReportObjectiveFiles' });
       ActivityReportObjective.hasMany(models.ActivityReportObjectiveTopic, { foreignKey: 'activityReportObjectiveId', as: 'activityReportObjectiveTopics' });
       ActivityReportObjective.hasMany(models.ActivityReportObjectiveResource, { foreignKey: 'activityReportObjectiveId', as: 'activityReportObjectiveResources' });
+      ActivityReportObjective.hasMany(models.ActivityReportObjectiveCourse, { foreignKey: 'activityReportObjectiveId', as: 'activityReportObjectiveCourses' });
+
+      ActivityReportObjective.belongsToMany(models.Course, {
+        through: models.ActivityReportObjectiveCourse,
+        foreignKey: 'activityReportObjectiveId',
+        otherKey: 'courseId',
+        as: 'courses',
+      });
```

Comment:
please move to activityReportObjectiveCourse.js

Reply:
done.

Reply:
This still needs to be moved